conftest.py

```diff
@@ -26,3 +30,17 @@ def isolate_environment():
     finally:
         os.environ.clear()
         os.environ.update(_environ)
+
+
+@pytest.fixture(scope="function", autouse=True)
+def isolate_logging():
+    "Ensure any changes to logging are isolated to individual tests" ""
+    _stdout = sys.stdout
+    _stderr = sys.stderr
+    try:
+        yield
+    finally:
+        sys.stdout = _stdout
+        sys.stderr = _stderr
```

Comment:
```suggestion
    try:
        yield
    finally:
        sys.stdout = sys.__stdout__
        sys.stderr = sys.__stderr__
```
[https://docs.python.org/3/library/sys.html#sys.\_\_stdout\_\_](https://docs.python.org/3/library/sys.html#sys.__stdout__)